source/opsmanual/creating-a-new-environment.html.md

```diff
@@ -0,0 +1,356 @@
+---
```

Comment:
@stephenharker should we remove this page? @SamLR and I think it's no longer useful.